notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -0,0 +1,1530 @@
+{
```

Comment:
<p>minor thing, with our timer we can do:</p><p> </p><p><code></code><code></p><p>with Timer() as time_train:</p><p>    ffm_model.fit(...) </p><p></code><code></code></p><p>to get the time you can do <code>time_train.interval</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
This is something implemented in NLP BP, right? Maybe worthing using it as a general function across different places.

Reply:
I will put this on hold and later on edit it once the generic functions are created. Is that okay?

Reply:
whatever you prefer, this function is in some notebooks in reco: https://github.com/microsoft/recommenders/blob/staging/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb. 

What are the generic functions?

Reply:
Ah I see. I missed that part. Will add it.